client/src/components/Common/CertificationItem/stories/index.story.js

```diff
@@ -12,9 +12,15 @@ storiesOf('Common/CertificationItem', module)
   .add('Not discovered', () => (
     <div style={{ maxWidth: '600px', padding: paddingSize }}>
       <CertificationItem
-        title={text('Title', 'This text will be hidden until user discovers it')}
```

Comment:
Here as well, probably is better to increase the allowed length of code for the line

Reply:
FIXED with the length of 100